src/store/modules/print.store.js

```diff
@@ -4,8 +4,23 @@ import log from '@/utils/logging.js'
 export default {
     state: {
         layouts: [],
+        selectedLayout: {},
+        selectedScale: 0,
+        printSectionShown: false,
+    },
+    getters: {
+        mapSize(state) {
+            var mapAttributes = state.selectedLayout.attributes.find((atr) => atr.name === 'map')
```

Comment:
might be good considering handling the case when `mapAttributes` is undefined (or has no `clientParams` etc...), maybe by returning `null` or `undefined` in these cases.

Also please switch to a `const` instead of `var`

Reply:
I already use `?`. It should be enough, to handle null or undefined cases. Right?